girder/api/describe.py

```diff
@@ -436,7 +436,7 @@ def __init__(self, templatePath=None):
         self.vars = {
             'apiRoot': '',
             'staticRoot': '',
-            'title': 'Girder - REST API Documentation',
+            'brandName': ModelImporter.model('setting').get(constants.SettingKey.BRAND_NAME),
```

Comment:
The bind right now will update the webroot at `/`, but not the API description page. You should be able to test and see that it's not updated until a restart.